src/isIsogram.test.js

```diff
@@ -6,4 +6,37 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true for an empty string', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it('should return true for words with mixed case and no repeating letters',
+    () => {
+      expect(isIsogram('isogram')).toBe(true);
+      expect(isIsogram('word')).toBe(true);
+      expect(isIsogram('abcdefghijklmnopqrstuvwxyz')).toBe(true);
+    });
+
+  it('should return true for words with repeating letters', () => {
+    expect(isIsogram('IsOgram')).toBe(true);
+    expect(isIsogram('WoRd')).toBe(true);
+  });
+
+  it('should return false for a word with repeating letters', () => {
+    expect(isIsogram('hello')).toBe(false);
+    expect(isIsogram('world')).toBe(true);
```

Comment:
The expected result for the test case 'world' should be false, as it has repeating letters when case-insensitive